lms/static/styles/components/_Dialog.scss

```diff
@@ -42,19 +42,20 @@
   @include focus.outline-on-keyboard-focus;
   border: 0;
   background: none;
-  color: var.$grey-6;
+  color: var.$grey-mid;
 
   // Given the button a large hit target and ensure the 'X' label is large
   // enough to see easily and aligned with the right edge of the dialog.
   // Add negative margins so that the button does not force the dialog to
   // grow in height.
   font-size: 20px;
+  font-weight: 600;
```

Comment:
beefed up the "X" button, It just looked too light to me and since the hover-color is no longer red, it needs more weight to be able to see that effect.